site/zenodo_rdm/legacy/deserializers/schemas.py

```diff
@@ -38,6 +73,26 @@ def _creators(self, data):
             ret.append(r)
         return ret
 
+    def _funding(self, grants):
+        """Transform funding information."""
+        _funding = []
+        for grant in grants:
+            # format "10.13039/501100000780::190101904"
+            id_parts = grant["id"].split("::")
```

Comment:
```suggestion
            id_parts = grant["id"].split("::", 1)
```

minor: just in case there's a grant that has for some reason a code with `::` (i.e. something like `10.1234/foobar::grant::code`)